### ISSUE TITLE
The problem of the LADAN_Topjudge_small's result

### ISSUE THREAD

**Issue by lvyougang** [prometheusXN/LADAN#7]
**I run this model for 16 epochs, but I can't get the accuracy which is written in the paper.
I used the original parameters.
Here are the Hyperparameters:**
epsilon=1e-9
batch_size = 128
max_epoch = 16
sent_len_fact = 100
doc_len_fact = 15
doc_len_law = 10
sent_len_law = 100
learning_rate = 1e-3

lstm_size = 128
clr_fc1_size = 512
clr_fc2_size = 256
law_relation_threshold = 0.3
vec_size = 200
shuffle = True

n_law = 103
n_accu = 119
n_term = 12

**Am I using the wrong parameters or is there something wrong somewhere?**

**Comment by lvyougang** [comment]
I found that the model has some randomness.
What is the seed  of the best model？
I want to reproduce your results. Thank you very much.
